keyboards/lets_split/keymaps/geripgeri/keymap.c

```diff
@@ -0,0 +1,438 @@
+ * |      |      |      |      |      |      |               |   4  |   5  |   6  |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |  VAI |  VAD |  SAI |  SAD | HUI  | HUD  |               |   3  |   2  |   1  |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |RGBMPR|RGBMFW|RGBTGL|               | Calc |   0  |   +  |      |      |      |
+ * `-----------------------------------------'               `-----------------------------------------'
+ */
+[_NUMPAD] = LAYOUT_ortho_4x12(
+  _______, _______, _______, _______, _______, _______,       KC_7,    KC_8,   KC_9,    _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       KC_4,    KC_5,   KC_6,    _______, _______, _______,
+  VAI_NEE, VAD_NEE, SAI_NEE, SAD_NEE, HUI_NEE, HUD_NEE,       KC_1,    KC_2,   KC_3,    _______, _______, _______,
+  _______, _______, _______, RGB_PRE, RGB_ANM, TOG_NEE,       KC_CALC, KC_0,   KC_PPLS, _______, _______, _______
+),
+
+/* Gaming
+ * ,-----------------------------------------.               ,-----------------------------------------.
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      | RAISE|      |               |      | LOWER|      |      |      |      |
+ * `-----------------------------------------'               `-----------------------------------------'
+ */
+[_GAMING] = LAYOUT_ortho_4x12(
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, RAISE,   _______,       _______, LOWER,   _______, _______, _______, _______
+)};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+	rgb_mode = RGBLIGHT_MODE_BREATHING + 1;
+	autoshift_enable();
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
+      break;
```

Comment:
Since you only have the one default layer, you can actually remove this, completely.